Dockerfile.assisted-service

```diff
@@ -5,6 +5,11 @@ COPY swagger.yaml .
 COPY tools/generate_python_client.sh .
 RUN chmod +x ./generate_python_client.sh && SWAGGER_FILE=swagger.yaml OUTPUT=/build ./generate_python_client.sh
 
+# TODO: Find a pure Python3 base image, rather than relying on the golang one
+FROM registry.ci.openshift.org/openshift/release:golang-1.15 as pybuilder
+COPY --from=swagger_py /build build
+RUN cd build && python3 setup.py sdist --dist-dir /assisted-service-client
+
 # Generate go client
```

Comment:
If we're removing the usage of `swagger_go` products, we can as well remove the generation of the products :)

Reply:
> If we're removing the usage of `swagger_go` products, we can as well remove the generation of the products :)

Not sure I understand

Reply:
If we don't use the generate go client anymore, we can just remove the generation itself.